lms/lmsweb/views.py

```diff
@@ -501,9 +501,13 @@ def _common_comments(exercise_id=None, user_id=None):
     Most common comments throughout all exercises.
     Filter by exercise id when specified.
     """
-    query = CommentText.filter(**{
-        CommentText.flake8_key.name: None,
-    }).select(CommentText.id, CommentText.text).join(Comment)
+    query = (
+        CommentText.select(CommentText.id, CommentText.text).join(Comment)
+        .join(User).join(Role).where(
+            CommentText.flake8_key.name is not None,
```

Comment:
According to [the documentation](http://docs.peewee-orm.com/en/latest/peewee/query_operators.html), this query might not work. Use `is_null` instead.